################[Necessary]################

## topic name (for sub)
# sending a message (with data=true) will trigger localization
trigger_topic_name: "/bbs_localize" ## msg type: std_msgs/Bool

map_topic_name: "/map/pointcloud_map" ## msg type: sensor_msgs/PointCloud2
lidar_topic_name: "/sensing/lidar/concatenated/pointcloud" ## msg type: sensor_msgs/PointCloud2
imu_topic_name: /sensing/imu/imu_data ## msg type: sensor_msgs/Imu

## topic name (for pub)
# note: PoseWithCovarianceStamped
# /initialpose is same topic when using rviz's 2d pose estimate func.
pose_topic_name: "/initialpose"

## 3D-BBS parameters
min_level_res: 1.0 # [m], e.g. outdoor: 1.0, indoor: 0.5
max_level: 6

## Angular search range [rad]
# 6.28 input is converted to 2*M_PI
min_rpy: [-0.02,-0.02,0.0] # [roll, pitch, yaw]
max_rpy: [0.02,0.02,6.28] # [roll, pitch, yaw]

## Score threshold
# score_threshold = floor(src_points.size() * score_threshold_percentage)
score_threshold_percentage: 0.9

################[Optional]################
## Downsample target(map) clouds [m]
map_leaf_size: 0.1 # off: 0.0

## Downsample source clouds [m]
src_leaf_size: 2.0 # off: 0.0

## Crop source clouds [m]
min_scan_range: 0.0 # [m]
max_scan_range: 100.0 # [m]
# off: set 0.0 to both min_scan_range and max_scan_range

## Timeout [msec] 
# Enter the number as int
timeout_msec: 0 # off: 0
 